### PR TITLE
JAMES-3400 Add remove and list domain aliases

### PR DESCRIPTION
**Pull request by chibenwa**
I had been playing a bit with the WebAdmin CLI, and added support for Domain Aliasses.

The road for a full CLI support with likely be long...

### CODE REVIEW

**Comment by chibenwa** [comment]
test this please

File: server/protocols/webadmin-cli/src/main/java/org/apache/james/cli/quota/SetGlobalQuotaCountCommand.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+    @CommandLine.Parameters
+    Long count;
+
+    @Override
+    public Integer call() {
+        try {
+            QuotaClient quotaClient = parentCommand.parentCommand.quotaCommand.fullyQualifiedURL();
+            Response rs = quotaClient.setQuotaCount(count);
+            if (rs.status() == DELETED_CODE) {
```
Comment (Arsnael) [inline server/protocols/webadmin-cli/src/main/java/org/apache/james/cli/quota/SetGlobalQuotaCountCommand.java:47]: DELETED_CODE for a set?

I see similar occurences as well... After rechecking the code, it seems those status code are generally declared locally in each class. However, I think they should be extracted and centralized in a proper class.

WDYT?

File: server/protocols/webadmin-cli/src/test/java/org/apache/james/cli/QuotaManageTest.java [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+                SoftAssertions.assertSoftly(softly -> {
+                    assertThat(exitCode).isEqualTo(0);
+                    assertThat(outputStreamCaptor.toString().trim().toCharArray()).containsOnly("256 bytes".toCharArray());
+                });
+            }
+        }
+    }
+
+    int executeFluent(String... args) {
```
Comment (Arsnael) [inline server/protocols/webadmin-cli/src/test/java/org/apache/james/cli/QuotaManageTest.java:204]: private?

**Comment by chibenwa** [comment]
:green_apple:  build

**Review by Arsnael — APPROVED** [review]

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**